src/applications/ivc-champva/shared/components/fileUploads/MissingFileOverview.jsx

```diff
@@ -199,7 +201,10 @@ export default function MissingFileOverview({
   requiredWarningHeading,
   showMail,
   mailingAddress,
+  mailPreamble,
   officeName,
+  requiredDescription,
+  optionalDescription,
   faxNum,
```

Comment:
Adding the three new text customization arguments (docstring above has been updated to reflect)